src/ui_menu_nbgl.c

```diff
@@ -35,10 +35,29 @@ enum {
 
 static nbgl_layoutSwitch_t switches[NB_SETTINGS_SWITCHES];
 
-static const char* const infoTypes[] = {"Version", "Developer", "Copyright"};
-static const char* const infoContents[] = {APPVERSION, "Vacuumlabs",
-                                           "(c) 2022 Ledger"
-                                          };
+static const char* const infoTypes[NB_INFO_FIELDS] = {"Version", "Developer", "Copyright"};
+static const char* const infoContents[NB_INFO_FIELDS] = {APPVERSION, "Vacuumlabs",
```

Comment:
Formatting can seem weird because tabs are used in this app, not spaces. But it's formatted with clang format.
@fbeutin-ledger @tdejoigny-ledger

Reply:
Indeed weird--> tab is not recommended

Reply:
@tdejoigny-ledger yeah I know but not sure we want to reformat the whole app... someone from vacuumlabs will have huge differences when wanting to merge with our branches (unless they reformat as well)